packages/amazonq/.changes/next-release/Bug Fix-85069edd-e299-43e4-b8b8-d1bfa7b635e0.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "filter out frequent client error from codegeneration telemetry"
```

Comment:
Not sure we really need a changelog item since this doesn't actually impact anything the customer sees/experiences